pkg/configutils/Write.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
we don't have any other go sources where the filename is title case like this

Reply:
Doh.